src/index.html

```diff
@@ -9,10 +9,45 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/index.css"
     />
   </head>
-  <body>
-    <h1>Calendar</h1>
+
+  <body class="page">
+    <h1 hidden>Calendar</h1>
+
+    <ul class="calendar calendar--start-day-sun calendar--month-length-31">
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
+      <li class="calendar__day"></li>
```

Comment:
Consider using a loop or a more dynamic way to generate the list items if possible. This will make the code more maintainable and reduce repetition.